script.js

```diff
@@ -1,11 +1,19 @@
 // Create an array called `colors` that contains three colors.
-
+var colors = ["red", "green", "blue"]
 // Log the last item in the array to the console.
-
+console.log(colors[colors.length - 1])
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+for (i = 0; i < colors.length; i++) {
+  console.log(`My favorite color is ${colors[i]}`)
+}
 // Create an `instructor` object that contains three key-value pairs.
-
+var instructor = {
+  name: "Adrian",
+  "hair-color": "brown",
+  attitude: "cool",
+  "facial-hair": "present"
+}
 // Add a `facial-hair` property to `instructor`.
 
```

Comment:
The intent of this question was to add the `facial-hair` property after the original object definition. So something like...

```js
instructor["facial-hair"] = true
```